tests/device_tests/ckb/test_get_address.py

```
@@ -0,0 +1,56 @@
+import pytest
```

Ruff (0.16.1)

[warning] The file is executable but no shebang is present

(EXE002)